src/Transport/AzureServiceBusTransport.cs

```diff
@@ -239,6 +251,6 @@ public RetryPolicy RetryPolicy
         }
         RetryPolicy retryPolicy;
 
-        internal string ConnectionString { get; private set; }
+        internal string ConnectionString { get; set; }
```

Comment:
Nitpicking: the class can be made partial and this property can be moved to the `obsoletes` file as a separate method so that all stuff to be removed is in a single place.

Reply:
not really, the ConnectionString is also used in the non-obsolete code-paths.